src/Sarif/Writers/BaseLogger.cs

```diff
@@ -23,9 +23,9 @@ public class BaseLogger
 
         private void ValidateParameters()
         {
-            if (resultKinds.Any(kind => kind != ResultKind.Fail))
+            if (resultKinds!=null && resultKinds.Any(kind => kind != ResultKind.Fail))
```

Comment:
>resultKinds!=null [](start = 16, length = 17)

is it needed? in theory i'm always initializing if the list received is null.

Reply:
You're right.  I missed that.  Fixed.

---
In reply to: [558733155](https://github.com/microsoft/sarif-sdk/pull/2246#discussion_r558733155) [](ancestors = 558733155)